readers/docker/containers.go

```diff
@@ -31,7 +31,7 @@ func (dc *DockerContainers) Run() error {
 
 	for _, container := range containers {
 		if container.ID != "" && container.NiceImageName != "" {
-			dc.Data[container.NiceImageName+"-"+container.ID] = container
+			dc.Data[container.Name] = container
```

Comment:
Are you sure that container.Name returns the nice tag name? Last time I pulled container.Name, it returned SHA.

That said, `&& container.NiceImageName != ""` needs to be changed to `&& container.Name != ""`